pkg/scheduler/framework/plugins/noderesources/resource_allocation.go

```diff
@@ -61,13 +61,13 @@ func (r *resourceAllocationScorer) score(
 	var score int64
 
 	// Check if the pod has volumes and this could be added to scorer function for balanced resource allocation.
```

Comment:
if you're removing the volume check, should you update this comment too?

Reply:
@damemi I suppose @draveness was trying to say the logic `>= 0` makes no sense as it's always true.

As mentioned in https://github.com/kubernetes/kubernetes/pull/95146#discussion_r496908079, checking `if len(volumes) > 0` might be the original intention.

Reply:
> @damemi I suppose @draveness was trying to say the logic `>= 0` makes no sense as it's always true.

yes

> As mentioned in [#95146 (comment)](https://github.com/kubernetes/kubernetes/pull/95146#discussion_r496908079), checking `if len(volumes) > 0` might be the original intention.

Thanks, I'll update the commit here.